design/sketch.webidl

```diff
@@ -574,16 +574,15 @@ enum GPUPrimitiveTopology {
 
 dictionary GPURenderPipelineDescriptor : GPUPipelineDescriptorBase {
     GPUPipelineStageDescriptor vertexStage;
-    GPUPipelineStageDescriptor fragmentStage;
+    GPUPipelineStageDescriptor? fragmentStage = null;
 
-    GPUPrimitiveTopology primitiveTopology;
+    GPUPrimitiveTopology primitiveTopology = "triangle-list";
     GPURasterizationStateDescriptor rasterizationState;
     sequence<GPUColorStateDescriptor> colorStates;
     GPUDepthStencilStateDescriptor? depthStencilState;
     GPUInputStateDescriptor inputState;
 
-    // Number of MSAA samples
-    u32 sampleCount;
+    u32 sampleCount = 1;
```

Comment:
MSAA off by default? This would be the opposite of WebGL.

Reply:
That's true, but I think it makes sense. It takes some extra effort to do multisample resolve in WebGPU - the canvas render target will always be nonmultisampled, so an explicit resolve is necessary. This makes it so the canvas render target can be used with a default render pipeline.